examples/AspNet/Global.asax.cs

```diff
@@ -73,8 +73,7 @@ protected void Application_Start()
             // https://github.com/open-telemetry/opentelemetry-dotnet/issues/2994
 
             var meterBuilder = Sdk.CreateMeterProviderBuilder()
-                 .AddAspNetInstrumentation()
-                 .AddHttpClientInstrumentation();
```

Comment:
Is this intentional?

Reply:
yes. HttpClient is not instrumented for .NET FW.